api/src/main/java/jakarta/mail/Session.java

```diff
@@ -1234,7 +1211,7 @@ private void loadAllResources(String name, Class<?> cl, StreamLoader loader) {
             URL[] urls;
             ClassLoader cld = null;
             // First try the "application's" class loader.
-            cld = getContextClassLoader();
+            cld = Thread.currentThread().getContextClassLoader();
```

Comment:
The original call swallowed SecurityException and returned null. Looking at other changes, we still handling SE except for getCCE.

Reply:
Seems I should clean those up as well.  I will update my branch to remove the SecurityException processing.  It was there because all I did was remove the surrounding with the doPriv block.

Reply:
The PR has been updated to include those changes.

Reply:
I was sort of debating on keeping the catches of SecurityException since a subclass of thread could just override  get/setContextClassLoader to unconditionally throw SecurityException even though SecurityManager doesn't exist.

Reply:
I based making the changes on the Javadoc for the methods which state SecurityException is thrown when using security manager.  If you think the catches should remain, I can redo it to put them back.

Reply:
If you would. I'm just a bit paranoid about SE getting thrown and changing the flow. Especially in the subclass thread case.

Reply:
I have updated to add back in the SecurityException catching for those API calls that could be from a subclass of Thread or ClassLoader.  I left the other SecurityException handling removed since those calls are on Java APIs that cannot be subclassed.

Reply:
Makes sense. Thanks for all the work on this!